html/interaction/spelling-and-grammar-checking/references/spelling-markers-001.html

```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE html>
```

Comment:
The "User interaction" section has ID "editing", and the paths are based on the ID, not the title. Move them all to `html/editing/editing-0/spelling-and-grammar-checking`, please.